### DISCUSSION TITLE
vrcx crashing

### DISCUSSION THREAD

**Discussion by julienthegamer** [vrcx-team/VRCX#409] [Q&A] [answered]
does anyone know why vrcx crashes whenever i start steam vr and try to open it in vr

**Comment by Natsumi-sama** [comment, marked as answer]
Could you try to disable "SteamVR Overlay" in VRCX Settings then restart VRCX to see if that fixes it.

**Reply by julienthegamer** [reply]
i wanted to have the overlay but when it tries to start it either lags me out then vrcx crashes or vrchat crashes

**Reply by Natsumi-sama** [reply]
Did it ever work for you on an older version?
Give these steps a try https://www.windowsdigitals.com/force-chrome-firefox-game-to-use-nvidia-gpu-integrated-graphics/

**Reply by julienthegamer** [reply]
thank you for helping me it has worked

**Reply by Natsumi-sama** [reply]
Well then, I didn't expect that to work but I'm glad it did.
